js_modules/dagster-ui/packages/ui-core/src/assets/AssetPartitionDetail.tsx

```diff
@@ -144,12 +152,34 @@ export const ASSET_PARTITION_DETAIL_QUERY = gql`
   ${ASSET_OBSERVATION_FRAGMENT}
 `;
 
+export const ASSET_PARTITION_STALE_QUERY = gql`
+  query AssetPartitionStaleQuery($assetKey: AssetKeyInput!, $partitionKey: String!) {
+    assetNodeOrError(assetKey: $assetKey) {
+      ... on AssetNode {
+        id
+        staleStatus(partition: $partitionKey)
+        staleCauses(partition: $partitionKey) {
```

Comment:
is this syntax correct? never seen it before :O

Reply:
Yeah I think this is actually two separate resolvers, both of which take a partition -- I think that it looks esp weird in this case because the first one just returns an enum value and there's no sub-selection